File: laravel-app/app/Models/Vote.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Concerns\HasUuids;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class Vote extends Model
{
    /** @use HasFactory<\Database\Factories\VoteFactory> */
    use HasFactory, HasUuids;
    protected $fillable = [
        'id',
        'poll_id',
        'option_id',
        'user_id',
        'voted_at',
    ];

    public function pollOption(): BelongsTo
    {
        return $this->belongsTo(PollOption::class, 'option_id');
    }
    public function poll(): BelongsTo
    {
        return $this->belongsTo(Poll::class, 'poll_id');
    }
}

File: netwinform-app/Models/Vote.cs
using System;
using System.Threading.Tasks;
using Newtonsoft.Json;


namespace netwinform_app.Models
{
    public class Vote
    {
        [JsonProperty("id")]
        public string Id { get; set; }
        [JsonProperty("poll_id")] 
        public string PollId { get; set; }
        [JsonProperty("option_id")]
        public string OptionId { get; set; }
        [JsonProperty("user_id")]
        public string UserId { get; set; }
        [JsonProperty("voted_at")]
        public string VotedAt { get; set; }
    }
}
